_includes/head.html

```diff
@@ -1,12 +1,80 @@
 
-  <link rel="stylesheet" href="{{ "/assets/chameleon/dist/css/chameleon.css" | relative_url }}">
-  <link rel="stylesheet" href="{{ "/assets/css/app.css" | relative_url }}">
-  <link rel="canonical" href="{{ page.url | replace:'index.html','' | absolute_url }}">
-</head>
+  <link rel="stylesheet" href="https://static.opensuse.org/chameleon/dist/css/chameleon.css" />
+  <link rel="stylesheet" href="{{ '/assets/css/app.css' | relative_url }}" />
+
+  <link rel='shortcut icon' type='image/x-icon'
+    href='https://static.opensuse.org/chameleon/dist/images/favicons/favicon.ico' />
+  <link color="#73ba25" href="https://static.opensuse.org/chameleon/dist/images/favicons/mask-icon.svg" rel="mask-icon">
+  <link href="{{ '/manifest.json' | absolute_url }}" rel="manifest">
+
+  <meta name="mobile-web-app-capable" content="yes" />
+
+  <meta property="og:site_name" content="{{ site.title | escape }}" />
+  <meta property="og:title"
+    content="{% if page.title %}{{ page.title | escape }}{% else %}{{ site.title | escape }}{% endif %}" />
+  <meta property="og:type" content="article" />
+  {% if page.layout == 'post' %}
+  <meta property="og:description" content="{{ page.content | strip_html | truncatewords: 50 }}" />
+  <meta property="og:article:author" content="{{ page.author }}" />
+  <meta property="og:article:published_time" content="{{ page.date | date_to_xmlschema }}" />
+  {% for tag in page.tags %}
+  <meta property="og:article:tag" content="{{ tag }}" />
+  {% endfor %}
+  {% endif %}
+  {% if page.image %}
+  <meta property="og:image"
+    content="{% if page.image contains '//' %}{{ page.image }}{% else %}{{ page.image | absolute_url }}{% endif %}" />
+  {% else %}
+  <meta property="og:image" content="https://static.opensuse.org/chameleon/dist/images/favicons/favicon.png" />
+  {% endif %}
+  <meta property="og:url" content="{{ page.path | replace:'index.html','' | absolute_url }}" />
+
+  <meta name="twitter:card" content="summary" />
+  <meta name="twitter:title"
+    content="{% if page.title %}{{ page.title | escape }}{% else %}{{ site.title | escape }}{% endif %}" />
+  {% if page.layout == 'post' %}
+  <meta name="twitter:description" content="{{ page.content | strip_html | truncate: 190 }}" />
+  {% endif %}
+  <meta name="twitter:url" content="{{ page.url | replace:'index.html','' | absolute_url }}" />
+  {% if page.image %}
+  <meta name="twitter:image"
+    content="{% if page.image contains '//' %}{{ page.image }}{% else %}{{ page.image | absolute_url }}{% endif %}" />
+  {% else %}
+  <meta name="twitter:image" content="https://static.opensuse.org/chameleon/dist/images/favicons/favicon.png" />
+  {% endif %}
+
+  <meta name="theme-color" content="#73ba25" />
+
+  <script defer src="https://static.opensuse.org/chameleon/dist/js/chameleon.js" type="text/javascript"
+    charset="utf-8"></script>
+
+  <!-- Piwik -->
```

Comment:
This piwik code is news-o-o specific

Reply:
Should we even keep it? In general, users don't want to be tracked, even if it is an open source project site. And, almost all ad blockers block it and show a red counter in browser toolbar...

Reply:
Here? No, we only include piwik where it matters on the case by case basis. news-o-o is one of those cases

Reply:
I added a boolean flag and `if` block. By default, no piwik script. It can be turned on if necessary.

Reply:
You will also need to setSiteId per website, so that should be a config option

Reply:
Added